invoke/runners.py

```diff
@@ -216,7 +216,7 @@ def run(self, command, **kwargs):
             err_stream = sys.stderr
         # Echo
         if opts['echo']:
-            print("\033[1;37m{0}\033[0m".format(command))
+            print(u"\033[1;37m{0}\033[0m".format(command))
```

Comment:
Windows is crazy: http://stackoverflow.com/questions/878972/windows-cmd-encoding-change-causes-python-crash/3259271
I'm still trying to get this work. As well as other bugs in this PR. It is not that easy as I thought at the beginning.